cmd/describe/machinepool/cmd_test.go

```diff
@@ -38,6 +38,7 @@ Tuning configs:
 Kubelet configs:                       
 Additional security group IDs:         
 Node drain grace period:               1 minute
+Management upgrade:                    
```

Comment:
How come it doesn't have the value for management upgrade?

Reply:
If there's no management upgrade, should we hide this instead of displaying?

Reply:
It will always have a value. The test nodepool doesn't have manageupgrade upgrade, but that's updated now.